tests/pytests/test_short_read.py

```diff
@@ -622,7 +628,7 @@ def doTest(env: Env, test_name, rdb_name, expected_index):
         sendShortReads(env, fullPath, expected_index)
 
 # Dynamically create a test function for each rdb file
-@skip(cluster=True, redis_less_than='6.2.0', macos=True, asan=True, arch='aarch64')
+@skip(cluster=True, redis_less_than='6.2.0', macos=True, asan=True)
```

Comment:
Any reason we used to skip on arm? Also why skipping on macos?

Reply:
IIRC, we coudnlt figure why it takes so long on MAC
I wanted to run it locally so i removed the restriction and it indeed got stuck (or i didn't wait long enough)
just forgot to revert 😅